<launch>
    <param name="robot_description"
           command="cat $(find learning_urdf)/urdf/r2d2.urdf" />
    <node name="robot_state_publisher"
          pkg="robot_state_publisher"
          type="state_publisher" />
    <node name="state_publisher"
          pkg="learning_urdf"
          type="state_publisher" />
</launch>